src/objecttypes/tests/commands/test_setup_configuration.py

```diff
@@ -0,0 +1,100 @@
+from io import StringIO
+
+from django.contrib.sites.models import Site
+from django.core.management import CommandError, call_command
+from django.test import TestCase, override_settings
+from django.urls import reverse
+
+import requests_mock
+from rest_framework import status
+
+from objecttypes.config.demo import DemoUserStep
+from objecttypes.config.objects import ObjectsAuthStep
+from objecttypes.config.site import SiteConfigurationStep
+
+
+@override_settings(
+    OBJECTTYPES_DOMAIN="objecttypes.example.com",
+    OBJECTTYPES_ORGANIZATION="ACME",
+    OBJECTS_OBJECTTYPES_TOKEN="some-random-string",
+    OBJECTS_OBJECTTYPES_PERSON="Some Person",
+    OBJECTS_OBJECTTYPES_EMAIL="objects@objects.local",
+    DEMO_CONFIG_ENABLE=True,
+    DEMO_TOKEN="demo-random-string",
+    DEMO_PERSON="Demo",
+    DEMO_EMAIL="demo@demo.local",
+)
+class SetupConfigurationTests(TestCase):
```

Comment:
Since you also test the behaviour per configuration step in their own tests, maybe you could mock out the `ConfigurationStep.configure` and just assert that it was called for each step? That way we don't test the same behaviour twice